playbooks/maas-restart.yml

```diff
@@ -16,6 +16,7 @@
 - name: Restart MaaS
   hosts: "{{ maas_restart_on_host_override | default('maas_restart_on_host') }}"
   gather_facts: false
+  serial: 1
```

Comment:
Can we make this configurable or using some percentage for serial instead of forcing all restarts to happen 1 at a time?

maybe using something like:

``` yaml
serial: "{{ maas_restart_serial | default(1) }}"
```

Reply:
Sure, I'll get that pushed out.